src/media/css/feed/collections.styl

```diff
@@ -35,10 +35,14 @@
     padding: 10px 10px 0;
 }
 
-[data-page-type~="homepage"] .collection-listing-header {
+.feed-home .collection-listing-header {
     bottom: 15px;
     right: 10px;
     width: unquote('calc(100% + 20px)');
+
+    +rtl() {
+        right: -10px;
```

Comment:
We'd have to unset the right then which implies two properties and I am fine with that but negative offsets don't worry me.